src/QMCHamiltonians/QMCHamiltonian.cpp

```diff
@@ -532,16 +532,16 @@ std::vector<QMCHamiltonian::RealType> QMCHamiltonian::flex_evaluate(const RefVec
       updateKinetic(HC_list[iw], H_list[iw], P_list[iw]);
     }
 
-    std::vector<RealType> local_energies(H_list.size(),0.0);
     for(int iw= 0; iw < H_list.size(); ++iw)
       local_energies[iw] = H_list[iw].get().get_LocalEnergy();
-    return local_energies;
-    
-    //}
-  // else if (H_list.size() == 1)
-  // {
-  //   H_list[0].get().evaluate(P_list[0]);
-  // }
+  }
+  else if (H_list.size() == 1)
+  {
+    std::vector<RealType> local_energies(1,0.0);
```

Comment:
Need resize, not declaration.

Reply:
There is resize above. Seems need nothing here.

Reply:
😢